python/caesar.md

```diff
@@ -207,7 +205,7 @@ This is my favorite one: enhance your program's "brute force" mode so that it ca
 
 <asciinema-player src="{{ site.baseurl }}/caesar_brute_smart_cast.json" rows="15" cols="80" autoplay="true" loop="true"></asciinema-player>
 
-You can do this however you want. Be creative! My solution involved using [this text file](https://svnweb.freebsd.org/csrg/share/dict/words?revision=61569&view=co), which is a list of all of the English words in the 1934 edition of Webster's Second International Dictionary. If you'd like to figure out how to open a text file in Python and get all the lines out of it, Google around until you find a solution - you can always ask me for help if you get stuck, but I think you'd be surprised how far you can get by just Googling stuff 🙂
+You can do this however you want. Be creative! My solution involved using [this text file](https://svnweb.freebsd.org/csrg/share/dict/words?revision=61569&view=co), which is a list of all of the English words in the 1934 edition of Webster's Second International Dictionary. If you'd like to figure out how to open a text file in Python and get all the lines out of it, Google around until you find a solution - you can always ask me for help if you get stuck, but I think you'd be surprised how far you can get by just Googling stuff. 🙂
```

Comment:
re: unrecognized character - ha, works on my machine! https://www.evernote.com/shard/s11/sh/ba89b6f8-6b58-438a-be55-ef7059f721df/ea381d04cc519079

i assume that when you were looking at this just now, you were on one of the computers in your classroom? if so, i'll just go ahead and change this to a regular :) smiley face - according to http://caniemoji.com/microsoft-windows/ , apparently chrome can't display emojis on windows < 10, which is surprising but oh well.

Reply:
Yep, I'm at school. It's a mac vs pc issue I bet...

Reply:
I get a lot of J's in my emails, which I think is the same issue but a different "unknown character" translation.

Reply:
yeah, that mystified me for years, i kept getting inexplicable Js in emails from business-type-y folks. i looked it up at one point and it turns out it's a bug in microsoft outlook. http://www.businessinsider.com/microsoft-outlook-fixes-j-smiley-emoticon-emoji-bug-2017-5 has details, sounds like they finally got around to publishing a fix for it seven years later, so hopefully that issue will go away over time as people update their copies of outlook!